gp-includes/advanced-permissions.php

```diff
@@ -49,7 +49,31 @@ function gp_allow_everyone_to_translate( $verdict, $args ) {
 	return $verdict;
 }
 
+/**
+ * Map the translation check to the translation-set
+ *
+ * @param  string|bool $verdict Previous decision whether the user can do this.
+ * @param  array       $args Permission details.
+ * @return string|bool New decision whether the user can do this.
+ */
+function gp_allow_approve_translation_with_validator_permissions( $verdict, $args ) {
```

Comment:
Is `gp_allow_approving_translations_with_validator_permissions` a better name for this function?